KarhooUISDK/CommonUI/BottomSheet/KarhooBottomSheet.swift

```diff
@@ -0,0 +1,165 @@
+//
+//  SwiftUIView.swift
+//  KarhooUISDK
+//
+//  Created by Diana Petrea on 03.11.2022.
+//  Copyright © 2022 Flit Technologies Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+struct KarhooBottomSheet<Content: View>: View {
+    // MARK: - Properties
+    private var viewModel: BottomSheetViewModel
+    @ViewBuilder private let content: () -> Content
+    @ObservedObject private var keyboard = KeyboardResponder()
+    @State private var offset = CGSize.zero
+
+    // MARK: - Init
+    init(viewModel: BottomSheetViewModel, @ViewBuilder content: @escaping () -> Content) {
+        self.viewModel = viewModel
+        self.content = content
+    }
+   
+    // MARK: - Body
+    var body: some View {
+        VStack(spacing: 0.0, content: {
+            Spacer()
+                .background(Color.clear)
+                .layoutPriority(Double(UILayoutPriority.defaultLow.rawValue))
+            
+            VStack(spacing: 0.0) {
+                HStack {
+                    Text(viewModel.title)
+                        .font(Font(KarhooUI.fonts.bodyBold()))
+                        .foregroundColor(KarhooUI.colors.text.getColor())
+                    Spacer()
+                        .background(Color.clear)
+                    Button {
+                        viewModel.dismiss()
+                    } label: {
+                        Image(
+                            uiImage:
+                                UIImage.uisdkImage("kh_uisdk_cross_new")
+                                .coloured(withTint: KarhooUI.colors.text)
+                        )
+                    }
+                    .frame(
+                        width: UIConstants.Dimension.Button.standard,
+                        height: UIConstants.Dimension.Button.standard
+                    )
+                }
+                .padding(
```

Comment:
In my opinion you should move this padding modifier to top most VStack (at least for leading and trailing). We want to have this paddings for all content inside bottom sheet and we should take care of it here, not inside any injected Content